src/status_im/transport/message/transit.cljs

```diff
@@ -0,0 +1,95 @@
+(ns status-im.transport.message.transit
```

Comment:
docstring and same with main public things here - central to understanding what abstractions we are talking about

Not clear what these tag and rep stuff are